src/GraphQL.NewtonsoftJson/InputsConverter.cs

```diff
@@ -6,7 +6,9 @@ namespace GraphQL.NewtonsoftJson
 {
     /// <summary>
     /// A custom JsonConverter for reading an <see cref="Inputs"/> object.
-    /// Doesn't support write.
+    /// Doesn't support writing, but since <see cref="Inputs"/> implements
```

Comment:
That is, control flow never falls into the `WriteJson` method?

Reply:
That is correct.